### ISSUE TITLE
bug: Cmdline expanding only in one direction when filled whole screen

### ISSUE THREAD

**Issue by superNWHG** [folke/noice.nvim#956]
### Did you check docs and existing issues?

- [X] I have read all the noice.nvim docs
- [X] I have updated the plugin to the latest version before submitting this issue
- [X] I have searched the existing issues of noice.nvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.10.1

### Operating system/version

Arch Linux

### Describe the bug

When using the cmdline of noice.nvim, it will expand when there is not enough space.
This works fine, until you try to expand it too much. When the size of the cmdline reaches the full width of the screen, it keeps expanding only on the right side. I think there is a border on the left side, that should also be on the right side.
This will lead to the "Cmdline" text at the top of the cmdline eventually moving out of the screen.

Demo video:

https://github.com/user-attachments/assets/2b75cb1a-8799-4e8c-9833-7394804faf54

### Steps To Reproduce

1. Use the noice.nvim plugin with the Cmdline enabled
2. Open the Cmdline
3. Type characters until you see the "Cmdline" text on top of the cmdline moving away

### Expected Behavior

The text staying at the right place and having a border on the right.

### Repro

```lua
vim.env.LAZY_STDPATH = ".repro"
load(vim.fn.system("curl -s https://raw.githubusercontent.com/folke/lazy.nvim/main/bootstrap.lua"))()

require("lazy.minit").repro({
  spec = {
    { "folke/noice.nvim", opts = {} },
    -- add any other plugins here
  },
})
```

**Comment by superNWHG** [comment]
Commenting to keep issue open since it's not resolved

**Comment by duguyue100** [comment]
Yep, this affects #970 which is a big bummer when creating new files.

**Comment by superNWHG** [comment]
Closing this since fixed in c49a4b0.
This seems to be the same issue as #959
